WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -113,6 +115,19 @@ public void setOutcome(Outcome newOutcome) {
         mOutcome = newOutcome;
     }
 
+    public void previewTemplate(String template) {
+        final Map<String, Object> properties = new HashMap<>();
+        properties.put(KEY_TEMPLATE, template);
+        AnalyticsTracker.track(Stat.EDITOR_SESSION_TEMPLATE_PREVIEW, properties);
+    }
+
+    public void applyTemplate(String template) {
+        mTemplate = template;
+        final Map<String, Object> properties = new HashMap<>();
+        properties.put(KEY_TEMPLATE, template);
```

Comment:
[Similar to iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/13510#discussion_r382498181), we should include `template` in the common properties and ensure all of those are attached in the preview and apply events. For preview, since we're not storing `template` into the session yet, we'd have to override that though.

Reply:
Updated